proto/commonfate/access/v1alpha1/access_request.proto

```diff
@@ -55,16 +56,25 @@ enum RequestStatus {
   REQUEST_STATUS_CLOSED = 3;
 }
 
+message Filter {
+  oneof filter {
+    commonfate.filters.v1alpha1.OccurredAtFilter occurred_at = 1;
+    commonfate.filters.v1alpha1.EntityFilter requested_by = 2;
+    commonfate.filters.v1alpha1.EntityFilter closed_by = 3;
+    commonfate.filters.v1alpha1.EntityFilter approved_by = 4;
+    commonfate.filters.v1alpha1.EntityFilter target = 5;
+    commonfate.filters.v1alpha1.EntityFilter role = 6;
+    commonfate.filters.v1alpha1.EntityTypeFilter target_type = 7;
+    commonfate.filters.v1alpha1.EntityTypeFilter role_type = 8;
+  }
+}
+
 message QueryAccessRequestsRequest {
   // The token for the next page.
   string page_token = 1;
   bool archived = 2;
   optional commonfate.entity.v1alpha1.Order order = 3;
-  optional commonfate.entity.v1alpha1.EID requested_by = 4;
-  optional commonfate.entity.v1alpha1.EID closed_by = 5;
-  optional commonfate.entity.v1alpha1.EID approved_by = 6;
-  RequestStatus request_status = 7;
```

Comment:
I think we need to keep the request status filter on here so we can do the 3 columns